src/backend/joanie/signature/backends/lex_persona.py

```diff
@@ -571,3 +571,63 @@ def get_signed_file(self, reference_id: str) -> bytes:
             )
 
         return response.content
+
+    def _prepare_update_of_signatories(self, reference_id: str) -> dict[dict]:
+        """
+        Prepare the payload to update an ongoing signature procedure.
+        """
+        validity_period = settings.JOANIE_SIGNATURE_VALIDITY_PERIOD
```

Comment:
```suggestion
        validity_period = settings.JOANIE_SIGNATURE_VALIDITY_PERIOD_IN_SECONDS * 1000
```

Reply:
Sorry, I've forgotten about the latest fix on the validity period for the signature. Good catch.